tests/drivers/display/display_read_write/testcase.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 (c) TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  tags:
+    - drivers display
```

Comment:
`tags: drivers display`
or
```
tags:
  - drivers
  - display
```
Otherwise there is just one tag "drivers display"

Reply:
I applied your suggestion.